src/components/TimetablePage/Listbox/index.tsx

```diff
@@ -69,7 +70,7 @@ function Listbox({
         })}
       >
         {value !== null ? list.find((item) => item.value === value)?.label : '학부'}
-        {version !== 'default' && <DownArrowIcon />}
+        {version !== 'default' ? <DownArrowIcon /> : <UpDownArrowIcon />}
```

Comment:
대체적으로 부정형 삼항 연산자를 긍정형으로 대체할 수 있다면 코드를 읽거나 통일하는 측면에서 수정하는 것이 좋을 것 같네요~!
```suggestion
        {version === 'default' ?  <UpDownArrowIcon /> : <DownArrowIcon /> }
```

Reply:
아하, 넵! 참고하여 수정하였습니다!
c8404370a8dc1072dfc1d99855b51da66c43b552